examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -129,6 +129,20 @@ export const TaskListView: React.FC<ITaskListViewProps> = (props: ITaskListViewP
 	const { taskList } = props;
 
 	const [tasks, setTasks] = useState<ITask[]>(taskList.getDraftTasks());
+	const [lastSaved, setLastSaved] = useState<number | undefined>();
+	const [failedUpdate, setFailedUpdate] = useState(false);
+	const handleSaveChanges = async (): Promise<void> => {
```

Comment:
I think we can make this just a `() => { ... }`. It's not doing any `await`ing so it doesn't need to be `async`, and if it's not async it can just return `void` (i.e. no declared return type) instead of a Promise.

Alternatively, await the call to `writeToExternalServer()` and wrap it with try/catch to handle the error, but I don't see a huge argument to do it that way over the current one.